<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestor ACOES: Generacion informe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="cTipo">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cMoneda">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calDateBegin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>304</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calDateEnd">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>140</y>
      <width>304</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cProyecto">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cApadrinamiento">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bCancelar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bGenerar">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>320</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Generar informe</string>
    </property>
   </widget>
   <widget class="QLabel" name="lTipo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Tipo</string>
    </property>
   </widget>
   <widget class="QLabel" name="lMoneda">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Moneda</string>
    </property>
   </widget>
   <widget class="QLabel" name="lProyecto">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Proyecto</string>
    </property>
   </widget>
   <widget class="QLabel" name="lApadrinamiento">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Apadrinamiento</string>
    </property>
   </widget>
   <widget class="QLabel" name="lFechaEmision">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Fecha de emision</string>
    </property>
   </widget>
   <widget class="QLabel" name="lAnteriorA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Posterior a:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lPosteriorA">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>120</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Anterior a:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../recursosQT.qrc"/>
 </resources>
 <connections/>
</ui>
